src/cljs/main/broadfcui/page/workspace/data/tab.cljs

```diff
@@ -218,10 +220,9 @@
                                                         [DataImporter {:workspace-id workspace-id
                                                                        :this-realm this-realm
                                                                        :import-type "data"
-                                                                       :reload
-                                                                       (fn [entity-type]
-                                                                         ((:request-refresh props))
-                                                                         (react/call :refresh (@refs "entity-table") entity-type))}])}]]]))
+                                                                       :do-this-afterwards
+                                                                       (fn []
+                                                                         (react/call :refresh (@refs "entity-table") (:selected-entity-type @state)))}])}]]]))
```

Comment:
this gives me red. isaac says it's because there's an outer hash-function and you can't nest them